ui/pages/swaps/index.js

```diff
@@ -127,11 +128,10 @@ export default function Swap() {
   const reviewSwapClickedTimestamp = useSelector(getReviewSwapClickedTimestamp);
   const reviewSwapClicked = Boolean(reviewSwapClickedTimestamp);
 
-  if (networkAndAccountSupports1559) {
-    // This will pre-load gas fees before going to the View Quote page.
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    useGasFeeEstimates();
-  }
+  // This will preload gas fees before going to the View Quote page.
+  useSafeGasEstimatePolling({ isEnabled: networkAndAccountSupports1559 });
+  // This will pre-gauge the network before going to the View Quote page.
+  useSafeNetworkCongestionPolling({ isEnabled: networkAndAccountSupports1559 });
```

Comment:
I wanted to write an e2e test to cover this behavior but Ganache doesn't support EIP-1559 transactions yet :(

Any idea on how we want to test this? The test file that matches this file is pretty sparse.